### PR TITLE
liveness returns "busy" if queued requests > available capacity

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
liveness returns "busy" if queued requests > available capacity

## Motivation and Context
applications relying on liveness can use this to 
ensure API availability on individual nodes

## How to test this PR?
Requires some manual work

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

File: cmd/healthcheck-handler.go [hunk 1]
```diff
@@ -106,6 +106,17 @@ func LivenessCheckHandler(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set(xhttp.MinIOServerStatus, unavailable)
 	}
 
+	if int(globalHTTPStats.loadRequestsInQueue()) > globalAPIConfig.getRequestsPoolCapacity() {
```
Comment (vadmeste) [inline cmd/healthcheck-handler.go:109]: Checking if len(t.requestsPool) == cap(t.requestsPool), if non zero, can work as well. You may like it better but still LGTM.

**Review by vadmeste — COMMENTED** [review]
One comment

**Review by klauspost — APPROVED** [review]

File: cmd/healthcheck-handler.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/healthcheck-handler.go:109]: IMO for readability this is what made sense to me @vadmeste

**Review by vadmeste — APPROVED** [review]
LGTM

**Pull request merged**